plugins/woocommerce/client/blocks/packages/components/form-step/style.scss

```diff
@@ -2,54 +2,52 @@
-	// e.g. Storefront using the "Default Template" (which has a sidebar) the checkout block can get the `.is-small`
-	// class, but the screen itself is larger than 400px.
-	@include breakpoint("<600px") {
-		padding-top: $gap-larger;
-
-		// Remove padding from the first checkout step. First of type is required here as the first-child is the express payment methods block.
-		&:first-of-type {
-			padding-top: 0;
-		}
-
-		&::after {
-			position: absolute;
-			content: "";
-			// Required because the box shadows are offset by 1px down so we need to move the element up by 1px.
-			bottom: calc(1px - $gap-larger);
-			width: 100%;
-			height: 1px;
-			opacity: 0.11;
-			background: currentColor;
-			// This box-shadow rule creates two visible shadows:
-			// - One 50vw to the left of the element.
-			// - Another 50vw to the right
-			// Both shadows are sharp (no blur) and use the current color of the element.
-			// Its purpose is to add a full-width divider between checkout sections which is otherwise impossible as
-			// the parent element has padding.
-			box-shadow: -50vw 0 0 0 currentColor, 50vw 0 0 0 currentColor;
-		}
-	}
-}
 .wc-block-components-form .wc-block-components-checkout-step {
 	position: relative;
 	border: none;
 	padding: 0;
 	background: none;
 	margin: 0 0 $gap-largest 0;
 
-	@include cart-checkout-mobile-container {
-		@include step-mobile-styles();
-	}
+	@include cart-checkout-below-medium-container {
+		padding-left: 0;
+		margin-bottom: $gap-larger;
+
+		// The below CSS rules show a separator between Checkout steps on mobile.
+		// This media query is required because on themes where the Checkout
+		// block is rendered alongside something else, e.g. Storefront using the
+		// "Default Template" (which has a sidebar) the checkout block can get
+		// the `.is-small` class, but the screen itself is larger than 400px.
+		@include breakpoint("<600px") {
+			padding-top: $gap-larger;
+
+			// Remove padding from the first checkout step. First of type is
+			// required here as the first-child is the express payment methods
+			// block.
+			&:first-of-type {
+				padding-top: 0;
+			}
 
-	@include cart-checkout-small-container {
-		@include step-mobile-styles();
+			&::after {
+				position: absolute;
+				content: "";
+				// Required because the box shadows are offset by 1px down so we
+				// need to move the element up by 1px.
+				bottom: calc(1px - $gap-larger);
+				width: 100%;
+				height: 1px;
+				opacity: 0.11;
+				background: currentColor;
+				// This box-shadow rule creates two visible shadows:
+				// - One 50vw to the left of the element.
+				// - Another 50vw to the right
+				// Both shadows are sharp (no blur) and use the current color of the element.
+				// Its purpose is to add a full-width divider between checkout
+				// sections which is otherwise impossible as the parent element has
+				// padding.
+				box-shadow: -50vw 0 0 0 currentColor, 50vw 0 0 0 currentColor;
+			}
+		}
 	}
```

Comment:
This is not removed; I refactored it a bit, and diff is doing a poor job showing what is going on 😅

Before:
```css
.wc-block-components-form .wc-block-components-checkout-step {
  // some styles here

  @include cart-checkout-mobile-container {
		@include step-mobile-styles();
	}

	@include cart-checkout-small-container {
		@include step-mobile-styles();
	}
```

After:
```css
.wc-block-components-form .wc-block-components-checkout-step {
  // some styles here

  @include cart-checkout-below-medium-container {
		// actual styles instead of @include step-mobile-styles()
	}
}
```